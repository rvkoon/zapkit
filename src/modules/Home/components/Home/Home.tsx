import { useTranslations } from "next-intl";

export interface IHomeProps {}

export function Home({}: IHomeProps): JSX.Element {
  const t = useTranslations("home");

  return (
    <div className="flex p-4">
      <div className="w-[400px] text-center m-auto flex flex-col gap-2">
        <h1 className="font-bold text-[24px]">{t("pageTitle")}</h1>
        <p>{t("pageDescription")}</p>
      </div>
    </div>
  );
}
