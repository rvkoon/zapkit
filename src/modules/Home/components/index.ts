export const HomeComponents = {};

export * from './Home';