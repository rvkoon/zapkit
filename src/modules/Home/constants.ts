export const Home_CONSTANTS = 'constants';