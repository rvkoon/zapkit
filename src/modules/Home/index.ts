export * from './components';
export * from './helpers';
export * from './hooks';
export * from './services';
export * from './types';
export * from './constants';