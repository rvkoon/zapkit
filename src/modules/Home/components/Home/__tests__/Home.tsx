import { Home, IHomeProps} from '../Home'
import { RenderResult } from '@testing-library/react'
import { render } from '@Tests'

describe("Home component", () => {
    function renderComponent(props: IHomeProps): RenderResult {
    return render(<Home {...props}/>)
    }

    it('should render correctly', () => {
    const { container } = renderComponent({})
    expect(container).toMatchSnapshot()
    })
})