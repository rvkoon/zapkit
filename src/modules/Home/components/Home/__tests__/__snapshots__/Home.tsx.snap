// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should render correctly 1`] = `
<div>
  <div
    class="flex p-4"
  >
    <div
      class="w-[400px] text-center m-auto flex flex-col gap-2"
    >
      <h1
        class="font-bold text-[24px]"
      >
        Welcome to ZapKit ðŸ‘‹
      </h1>
      <p>
        ZapKit is a NextJs starter kit featuring tailwindcss, typescript, i18n, jest, tanstack, and more. 
 It's gonna help you starting your new project ultra fast, without thinking about config concerns.
      </p>
    </div>
  </div>
</div>
`;
